### ISSUE TITLE
[suggestion] Missiles with AOE explosives or EMP

### ISSUE THREAD

**Issue by egilburg** [endless-sky/endless-sky#423]
A missile which you fire, and upon hitting an enemy, it bursts an explosion and damages multiple nearby enemies.

It would have 2 major uses:

1. Used by the player it would be useful for a maneuver warfare against lots of enemies - stay on the edges, kite big groups of enemies, and attenuate them with shrapnel.

2. Used by NPCs, it would be a counter to a "big ball of doom" tactic where you can just buy dozens of heavy warships and obliterate your way with brute force. AOE used by enemies would be devastating to large clustered fleets that don't bother to dodge or maneuver, and would allow numerically disadvantaged enemies to protect systems against a big clustered player fleet, forcing the player to fight a battle of strategy and maneuver rather than just stomp over everything with a dozen or more heavy warships with no regard to tactics. 

AOE weapons could also have an EMP variant which mass-disables rather than destroys enemies. This would be very useful as an emergency escape if chased by a large fleet of enemies, or as a crowd control tactic where you can temporarily disable a bunch of ships while you focus on destroying a single one.

**Comment by thenerdfreak** [comment]
Heavy Rocket works like this

**Comment by egilburg** [comment]
Ok gotta try it out - if it does I feel NPCs should be using it more often.

IMO a big blob of doom should be *severely* penalized with AOE in strong militarized systems, it feels too easy currently.

**Comment by thenerdfreak** [comment]
Mostly smal pirates use them

**Comment by egilburg** [comment]
>Mostly smal pirates use them

Which itself feels wrong, pirates are mostly independent hunters who expect to fight single enemies, massive AOE wouldn't be useful to them. It's big militiaries protecting core systems which I think should be equipped with devastating AOEs, specifically to be able to hold against a massive enemy fleet invasion.

**Comment by thenerdfreak** [comment]
Nah they do enough damage to take small interceptors in two shots
that's why pirates use them

**Comment by Pointedstick** [comment]
The Free Worlds operate a lot of Bastion warships that are armed with Heavy Rockets, which makes sense given the amount of light pirate Interceptor-class ships operate in the vicinity. I think what you're asking for is already the case.

**Comment by endless-sky** [comment]
The engine would need to be adjusted to allow weapons with proximity
triggers to fire off their submunitions instead of just exploding, but the
rest of this is already possible with the current engine.

**Comment by anonybosch** [comment]
AOE?

**Comment by egilburg** [comment]
Area-of-effect, i.e. explosion that damages all enemies within a certain radius without needing to him them directly.

**Comment by brendanjones** [comment]
The variance in the missiles in EVN are a great model, I think. They made the game a great deal more interesting with their differences in speed, tracking, rate of fire, AOE effect and damage type (shields or armour). Think the fast, weaving Hellhound missile with its AOE damage and massive impact, and the slow moving EMP Torpedo with its massive shield damage. ES needs a bit of this variance to spice things up and force changes in strategy dependent on what foe/weapons you're facing.

Missiles are currently pretty much useless against the player in ES once you've got even a couple of Bacs with Heavy Anti-Missle Turrets, let alone Korath/Pug/Quarg anti missile outfits. IMO anti-missile turrets are OP in ES.

TL;DR I agree, some missiles with AOE would be a great addition, along with some more variance in the other attributes of missiles.

**Comment by endless-sky** [comment]
See #346 - a big change to missiles that I'm planning on making as soon as
the "0.8" stable series of releases is done and it's time to move on to the
"0.9" experimental series, is to have different kinds of missile tracking
that can be defeated in different ways. For example, some missiles might
track heat, others might track mass, and others might use radar (which
could be jammed). That will allow for a much wider variety among the
missile weapons.

>

**Comment by Pointedstick** [comment]
I greatly look forward to that change, @endless-sky. Differentiation will make missiles much more interesting. Torpedoes should be slower and clumsier but more powerful and harder to jam with multiple guidance methods. Sidewinders should be faster, more agile, harder to destroy, and harder to jam, but with a smaller payload. Meteors should be cheap and average, but relatively easy to jam.

**Comment by brendanjones** [comment]
yay! Great work @endless-sky .
